connectors/preflight_check.py

```diff
@@ -92,8 +92,10 @@ async def _check_system_indices_with_retries(self):
         while self.running:
             try:
                 # Checking the indices/pipeline in the loop to be less strict about the boot ordering
-                await self.es_client.check_exists(
-                    indices=[CONNECTORS_INDEX, JOBS_INDEX]
+                # Using concrete write index to create these to ensure ES-installed template takes
+
+                await self.es_client.ensure_exists(
+                    indices=[CONNECTORS_INDEX + '-v1', JOBS_INDEX +'-v1']
```

Comment:
Could you please add a comment on why `-v1` is being used? Or maybe move the aliases to the constants?

Reply:
There's a comment there! We're using the concrete write index because the template is triggered when targeting the concrete write index. We cannot use an alias to create the index (the alias is created by the template!), but we can use it for everything else.